specs/airr-schema.yaml

```diff
@@ -1954,24 +1973,60 @@ Clone:
         deletions:
             type:  string
             description: Clone deletion mutations
+        CloneRepertoireList:
+            type: array
+            description: Properties of the clone members in each Repertoire
+            items:
+                - $ref: '#/CloneRepertoire'
+        TreeList:
+            type: array
+            description: Trees describing the phylogenetic structure of the clone.
+            items:
+                - $ref: '#/Tree'
 
-# Clone information for a specific repertoire within a single clone processing.
+# Clone information within a single repertoire
+# 1-to-n relatonship between Clone and Clone Repertoire
 CloneRepertoire:
     discriminator: AIRR
     type: object
     properties:
         clone_id:
             type: string
             description: Identifier for the clone.
-        data_processing_id:
-            type: string
-            description: Identifier for the data processing object.
+            $ref: '#/Clone'
         repertoire_id:
             type: string
             description: Identifier for the repertoire object.
+        rearrangements:
+            type: array
+            description: List of rearrangments that are members of this Clone in this Repertoire
+            items:
+                allOf:
+                    - $ref: '#/RearrangementSet' #I think we don't need this yet...
+                    - $ref: '#/Rearrangement'
+        #some examples copied from the immunedb schema
+        instances:
+           type: integer
+           description: Clone overall instance count 
+        avg_v_identity:
+            type: string
+            description: The average V-gene NT identity across all the sequences in a clone.
+        top_copy_seq_sequence:
+            type: string
+            description: The full aligned sequence with the highest copies in the clone
+        top_copy_seq_copies:
+            type: integer
+            description: The number of copies for top_copy_seq_sequence
+        top_instance_seq_sequnence:
+            type: string
+            description: The sequence for instance with the highest copies
+        top_instance_seq_copies:
+            type: integer
+            description: The number of copies for top_instance_seq_sequence
 
 # 1-to-n relationship for a clone to its trees.
-CloneTree:
+# Is there a reason to call this 'CloneTree' instead of just 'Tree'?
```

Comment:
Tree seems good from an Olmsted perspective and in general since it is a "clones and trees (of clones)" schema